translations/lxqt-about_de.ts

```diff
@@ -24,7 +24,7 @@
         <location filename="../aboutdialog/lxqtaboutdialog.cpp" line="100"/>
         <source>Copyright: © %1-%2 %3</source>
         <comment>About dialog, &apos;About&apos; tab text</comment>
-        <translation>Copyright: © %1-%2 %3</translation>
+        <translation>Urheberrecht: © %1-%2 %3</translation>
```

Comment:
@OliverBurkardt - ok, wieder so ein Fall, den man dokumentieren sollte - Copyright solle man wohl nicht übersetzen - ich hab auf jeden Fall ein ungutes Gefühl dabei.

In diesem Zusammenhang - Wir sollten, wenn wir mal viel zu viel Zeit haben - diesen GPL-Scheiss aufräumen. Und das mit dem Scheiss mein ich wörtlich - (L,A)GPL und anderes Lizenzzeug einfach nicht übersetzen. Und wenn man das wirklich übersetzen wollte, dann die offizielle Übersetzung nehmen (auch Snippets) - nen Link würde es meist auch tun.

Reply:
Wir sind uns einig, dass offizielle Übersetzungen zu verwenden werden sollten, im Bezug auf rechtliche Sachverhate, wie Lizenzen.

Ich habe keine Quellen finden können in denen es untersagt wäre "copyright" zu übersetzen. Nach meiner Recherche "Urheberrecht" der offiziell anerkannte deutsche Begriff für den englischen Begriff "copyright". Die Desktopumgebung XFCE schreibt ebenfalls durchgängig von "Urheberrecht" in der deutschen Version. Nicht  englischsprachige Menschen sollen ja verstehen um was es sich handelt. Wenn jemand "copyright" nicht versteht, weil er kein Englisch kann (sicherlich ein Ausnahmefall), macht es keinen Sinn "copyright" zu schreiben.

Ich lasse "Urheberrecht" deshalb noch drin, als "to be reviewed". Das kommt auch in die Dokumentation.

Reply:
Ok, noch mal überdenken - um das Urheberrecht will ich mich nicht wirklich streiten - um die Cursor-Tasten schon :sunglasses: